config/compareWarpAssembleCoadd.py

```diff
@@ -1,7 +1,8 @@
 config.badMaskPlanes = ("BAD", "EDGE", "SAT", "INTRP", "NO_DATA",)
-config.subregionSize = (10000, 200) # 200 rows (since patch width is typically < 10k pixels
+config.subregionSize = (10000, 200)  # 200 rows (since patch width is typically < 10k pixels
 config.doMaskBrightObjects = True
+config.removeMaskPlanes.append("CROSSTALK")
```

Comment:
Slightly confusing on this ticket, but doesn't really matter.

Reply:
Was hoping that the separate commit and  message made it clear it is something missed from DM-12665.